### DISCUSSION TITLE
Kafka via SSL configuration without Docker

### DISCUSSION THREAD

**Discussion by DyubVlad** [provectus/kafka-ui#3084] [Q&A] [answered]
I run the application locally without Docker using the command:
java -Dspring.config.additional-location=<path-to-application-local.yml> -jar <path-to-kafka-ui-jar>
Can you suggest what should be written in application-local.yml to connect to Kafka via SSL?

**Comment by Haarolean** [comment, marked as answer]
Hi,
[here's an example](https://github.com/provectus/kafka-ui/blob/master/documentation/compose/kafka-ssl.yml). Every underscore is a newline indent in yaml, you get the idea.
